review_process.md

```diff
@@ -1,68 +1,67 @@
-- scoping out deliverables and timelines in greater details
+Upon submission, the *Granting Entities*, 
+[Interchain GmbH.](https://interchain.berlin/) and 
+[Informal](https://informal.systems/), will assess whether an
+application is complete and meets the legal criteria 
+for consideration.
 
-with the goal to assembling a Grant or Service Agreement between the ICF and
-the application.
+Granting Entities will then perform an initial review using the
+following [evaluation criteria](./evaluation_criteria.md),
+and solicit additional peer review where appropriate.
 
-**NOTE:** At this point, no funding has yet been approved. Some
-back and forth may occur between the first call and next step.
+Each proposal will then be matched with a *Steward*, who will serve
+as the Applicant's primary point of contact.
 
-## Approval by the Board of Management (BoM)
+Once assigned, Stewards will setup an initial call with the Applicant
+to clarify outstanding questions about their submission and revise
+the proposed scope, timeline, and budget as needed. Together they 
+will draft a prospective *Financing Agreement*, which will used as 
+the basis for subsequent evaluation.
 
-At some point in the previous step, the BoM will have given unanimous
-approval to fund certain projects, for which an Executive Summary is
-created. All the projects approved by the BoM will then be assembled
-in a package sent to the Foundation Council for review. This Executive
-Summary can be found at the bottom of the [Evaluation Criteria](./evaluation_criteria.md).
+## Board of Management & Technical Advisory Committee
 
-## Recommendation to the Foundation Council
+The draft Financing Agreement, together with Granting Entity reviews
+and the original application are delivered to the *Board of
+Management*, a supervisory body employed by the Interchain Foundation.
 
-The Foundation Council reviews each project and either approves or
-denies the funding. Given the due diligence that has occurred thus far,
-the FC must give good reasons for denying funding.
+The Board of Management will administer the assembly of a *Technical
+Advisory Committee*, composed of domain experts with a record
+of community contribution. This committee will provide feedback
+on each proposal's technical soundness and benefit to the wider
+ecosystem.
 
-## Approval by the Foundation Council
+The Board of Management will then incorporate feedback from the 
+Technical Advisory Committee with all previous materials, to create
+a final package which is either unanimously endorsed by the Board of
+Managment or returned to Stewards with revising comments.
 
-Once the Foundation Council approves the funding, agreements are drafted and finally signed.
+## Foundation Council
 
-## Timeline
+Endorsed packages are then sent to the *Foundation Council*,
+who will certify their final approval by signing the 
+Financing Agreement, or exercising a final veto. Given the 
```

Comment:
they don't actually sing anything here. they just vote to approve the budget, the financing agreement often actually doens't exist at this point. I think it's good to say we'll submit the financing agreement to FC as the details should be ironed out at this point, but it's not the FC signing it necessarily.

Reply:
Hmm, another version gave the impression that steward worked together with applicant to come up with a draft financing agreement that was then passed along to FC. I guess this isn't the case though. When is financing agreement created?

Reply:
removed signing language:

> Endorsed application packages are then sent to the Foundation Council, who will either give final approval or exercise their veto.

Reply:
after it's approved Michael will work with the applicant to finalize the financing agreement. There they can negotiate payment details like atoms v fiat and cadence etc.